openrcv3_domain

```diff
@@ -12,7 +12,7 @@ else
 fi
 _password=$(juju run $_juju_model_arg --unit keystone/leader 'leader-get admin_passwd')
 
-if [ -s $_root_ca ]; then
+if [ ! -v $_root_ca -a -s $_root_ca ]; then
```

Comment:
`-v` isn't valid for `[`
`[` is an alias for test, and https://linux.die.net/man/1/test doesn't contain `-v`